include/color.h

```diff
@@ -21,4 +21,22 @@ typedef struct {
     f32 r, g, b, a;
 } Color_RGBAf;
 
+typedef union {
+    struct {
+        u16 r : 5;
+        u16 g : 5;
+        u16 b : 5;
+        u16 a : 1;
+    };
+    u16 rgba;
+} Color_RGB5A1;
+
+typedef union {
+    struct {
+        u8 a : 3;
+        u8 b : 5;
+    };
+    u8 c;
+} Struct_Cvg;
```

Comment:
This doesn't seem to be used anywhere?

Reply:
This was part of a matching idea that didn't go anywhere. I'll take it out.